src/main/java/eu/bbmri/eric/csit/service/negotiator/configuration/mappers/PostModelMapper.java

```diff
@@ -0,0 +1,32 @@
+package eu.bbmri.eric.csit.service.negotiator.mappers;
+
+
+import eu.bbmri.eric.csit.service.negotiator.database.model.Person;
+import eu.bbmri.eric.csit.service.negotiator.dto.person.PersonDTO;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.TypeMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class PostModelMapper {
+
+  @Autowired
+  ModelMapper modelMapper;
+
+  void addMappings() {
```

Comment:
Could you add a test for Post maapings?

Reply:
Fixed in 9e79a341c22c9d82b6f49666b7b974893587a33c